pkg/operator/ceph/object/user/controller_test.go

```diff
@@ -253,6 +253,9 @@ func TestCephObjectStoreUserController(t *testing.T) {
 		TypeMeta: metav1.TypeMeta{
 			Kind: "CephObjectStore",
 		},
+		Status: &cephv1.ObjectStoreStatus{
```

Comment:
The unit test doesn't really verify that it's set. How about a simple check in an integration test that the status is set as expected?

Reply:
Done in `pkg/operator/ceph/object/controller_test.go`.